### DISCUSSION TITLE
SimNIBS GUI Error

### DISCUSSION THREAD

**Discussion by Dore95** [simnibs/simnibs#183] [6. Getting started]
Hello,

When I open SimNIBS after downloading it for the first time, I get the following error:
![IMG20230814153019](https://github.com/simnibs/simnibs/assets/142486704/d5d3852d-bf6c-4249-87a0-070cabf59e35)

Is this familiair to anyone? Or does anyone know how to handle this?

Thank you in advance.

Kind regards,
Dore

**Comment by oulap** [comment]
We had one other user report this error, but we did not solve it yet. Could you try to update the PyOpenGL package shipped with simnibs and see if that helps? You can do that by opening a terminal and calling:

simnibs_python -m pip install PyOpenGL --upgrade

and then running the GUI again.
